app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2><%= @recipe.name %></h2>
+<p>Complexity: <%= @recipe.complexity %></p><br>
+<p>Genre: <%= @recipe.genre %></p><br>
+<h3>Ingredients</h3>
+<section id="ingredients">
+<% @ingredients.each do |ingredient| %>
+  <p><%= ingredient.name %></p>
+<% end %>
+</section><br>
+<h3>Total Cost: <%= @recipe.total_cost %></h3>
+<h4>Add an Ingredient</h4>
+<%= form_with url:"/recipes/#{@recipe.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
A slightly more RESTful path would be `/recipes/:recipe_id/ingredients` - just to indicate that you're dealing with the ingredients for a recipe